index.html

```diff
@@ -8,7 +8,7 @@
     />
     <script
       type="module"
-      src="/src/main.ts"
+      src="/src/main.js"
```

Comment:
Using TypeScript should be preferred for the type safety
```suggestion
      src="/src/main.ts"
```

Reply:
Done